test/unit/range/container/constexpr_string_test.cpp

```diff
@@ -132,7 +132,9 @@ TEST(constexpr_string, concat)
         constexpr_string em = constexpr_string{"hello"} +
                               constexpr_string{' '} +
                               constexpr_string{"world"};
-        EXPECT_EQ(c_foo(em.size()), 11u);
+        constexpr auto size = em.size();
+
```

Comment:
Please put the empty line above `constexpr auto size = em.size();`

Reply:
Why? In a block I declare all variables and in the second block I assert the values